charters/QueryService/charter.md

```diff
@@ -0,0 +1,77 @@
+# Query Service
+
+## Description
+The Query Service is an Application Programming Interface (API) for querying Human Cell Atlas (HCA) bundle metadata with intuitive query syntax.
+
+Services the Query Module provides may complement or subsume the current search functionality in the Data Storage Service (DSS).
+
+## Objectives
+1. Build, operate, and maintain a metadata API query service that allows a variety of queries to be performed against bundle metadata stored in HCA Data Coordination Platform (DCP) DSS
+1. Develop a query language specification for intuitive querying of HCA metadata
```

Comment:
Are these all '1's on purpose? i.e. is it indicating priority or is it supposed to be a numerical list?

Reply:
Ooh, today I learned! Thanks!